syntax = "proto3";
package jio.brain.proto.knowledge.healthcare;

import "jio/brain/proto/knowledge/healthcare/base/attribute.proto";
import "jio/brain/proto/knowledge/healthcare/base/status.proto";

message GetAttributesOfSymptomAndDiseaseRequest {
    uint32 symptom_id = 1;
    repeated uint32 disease_id = 2;
}

message GetAttributesOfSymptomAndDiseaseResponse {
	repeated jio.brain.proto.knowledge.healthcare.base.Attribute attributes = 1;
    jio.brain.proto.knowledge.healthcare.base.Status status = 2;
}